(DOCSP-22032): Add details for Swift SDK offline login (#1846)
mongodb/docs-realm: source/includes/offline-login.rst

When your Realm application authenticates a user, it caches the user's 
credentials. You can check for existing user credentials to bypass the 

> login flow and access the cached user. Use this to open a {+realm+} offline. 

Reviewer: i think this bit should just be part of the swift doc as opposed to include, so you can link to the docs about logging in user offline.

Author: I didn't feel the need to link the user to the other page from here because it's literally the exact same information - same include, same code example, but the open a realm page has a little extra info about downloading a realm blocking the opening a realm offline functionality. (Although one could argue that should be on both pages if the primary use for logging in a user "offline" is to open a realm.)

But there's no additional info in the other place, so it seems unnecessary to link there if it doesn't give the user more info.

Reviewer: ah ok fair nuff. i didn't compare the code examples

> .. note:: A user must have a network connection for first login

Reviewer: ```suggestion
.. note:: A user must have a network connection to sign up
```

>    The first time the user logs in, they must have a network connection.

Reviewer: ```suggestion
   When a user signs up for your app or logs in for the first time with an existing account on a client,
   they must have a network connection.
```
